### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by thok54**
Still can´t mark directories as Root. Whenever I click on "Mark as...", only option is "Excluded"

### CODE REVIEW

File: src/main/java/ejercicio/banco/service/AccountServiceImpl.java [hunk 1]
```diff
@@ -36,4 +36,20 @@ public Account findAccount(int n) {
         return account;
     }
 
+
+    //Store user
```
Comment (jantpedraza) [inline src/main/java/ejercicio/banco/service/AccountServiceImpl.java]: What is the point of this comment? The method is super clear, `storeAccount`. 

As general, a good rule for the javadocs is: Only describe what is not clear enough with the method name + the received parameters.

File: src/test/java/ejercicio/banco/tester/CsvAccountRepositoryTest.java [hunk 2]
```diff
@@ -29,6 +31,23 @@ public void findTest(){
 
     }
 
+    @Test
+    public void otherTests(){
```
Comment (jantpedraza) [inline src/test/java/ejercicio/banco/tester/CsvAccountRepositoryTest.java]: OMG!! `otherTests()`!!! Never again please :)

Use a proper method for the tests, this is super important.

**Review by jantpedraza — CHANGES_REQUESTED** [review]

**Comment by thok54** [comment]
Sorry If I have not been answering to each comment individually. By the time I see them, there are various of them so I try to answer them all by uploading the requested changes together.

**Comment by thok54** [comment]
I think I am not sure I understand the exercise.

Am I suppose to have a single generate method that can produce either of our 3 objects, or do we want an independent implementation for each of the 3 classes?

Also, I am supposed to tell what kind of item I "have" in order to call the proper method.
What object am I suppose to be checking?

Would it not be much simpler to simply have something like: "generateRandomBank" when I want a random bank?

**Comment by jantpedraza** [comment]
Te viene bien mañana por la noche un skype y lo hablamos? Sobre las 21:00 hora Málaga. Sino, ve implementando la estructura del buscador y las clases y luego lo vemos.

Enviado desde mi iPhone

**Comment by jantpedraza** [comment]
Estoy pensando Jesús, que para simplificar el problema, vamos a generar aleatoriamente los bancos; payments y accounts, pero el buscador sólo va a buscar en nombres de bancos y cuentas, ok? De forma que será un string.

Si ves que el factory no consigues sacarlo, hazlo cómo tu dices y luego vemos el porqué de usar ese patrón.

Enviado desde mi iPhone

**Review by jantpedraza — APPROVED** [review]

**Pull request merged**